spec/latest/json-ld/index.html

```diff
@@ -1043,6 +1043,12 @@
     underscore (<code>_</code>), the value is interpreted as <a>blank node identifier</a>
     instead.</p>
 
+  <p class="changed">In JSON-LD 1.1, terms will be used as compact IRI prefixes
+    when compacting only if
+    they are <a>simple term definitions</a> and either end with a URI <a data-cite="RFC3986#section-2.2">gen-delim</a> character (e.g,
```

Comment:
The term doesn't end with a URI `gen-delim` the value (string) it maps to does... so a slight rewording here and the few other places in the spec this was copied to:

`terms will be used as compact IRI prefixes when compacting only if a <a>simple term definition</a> is used and its value ends with a URI ...`